services/webhook/pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+authors = ["Sylvain Lesage <sylvain.lesage@huggingface.co>"]
+description = "REST API app"
+name = "webhook"
+version = "0.1.3"
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "3.9.18"
+environs = "^9.5.0"
+jsonschema = "^4.17.0"
+libapi = {path = "../../libs/libapi", develop = true}
+soundfile = ">=0.12.1"
```

Comment:
not sure we need this (same in the docker file) but...

Reply:
I remove it